Merge extension package into core one (#545)
appsignal/appsignal-nodejs: packages/nodejs/.changesets/merge-extension-package-into-nodejs.md

---
bump: "minor"
type: "change"
---

> The extension package (@appsignal/nodejs-ext) is no longer a separated dependency. Now, it is
> merged into @appsignal/nodejs.

Reviewer: ```suggestion
The extension package (@appsignal/nodejs-ext) is no longer a separated dependency. The extension is part of the @appsignal/nodejs package now.
```
